### PR TITLE
[CP-1090] Implementing infinity loading for large contacts lists

### PR DESCRIPTION
**Pull request by igorbezsmertnyi**
Jira: [CP-1090]

**Description**
Implementing infinity loading for large contacts lists

<details>
<summary><b>Screenshots</b></summary>
// put images here
</details>

[CP-1090]: https://appnroll.atlassian.net/browse/CP-1090?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by wojcik92michal — CHANGES_REQUESTED** [review]
Rejecting due to the fact that contacts search is now broken.

**Review by wercix — DISMISSED** [review]

**Review by wercix — COMMENTED** [review]
Looks good, search is working. But now after checking a checkbox the list is loaded one more time. Can we improve this experience?

**Comment by igorbezsmertnyi** [comment]
@wercix https://github.com/mudita/mudita-center/pull/966/commits/da92d060b575c56715c7b41cc0aec8e29fd86164

**Comment by igorbezsmertnyi** [comment]
@wojcik92michal

File: packages/app/src/contacts/components/contact-group/contact-group.styled.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/**
```
Comment (wojcik92michal) [inline packages/app/src/contacts/components/contact-group/contact-group.styled.ts:1]: (adding it here as it is easier to have a thread)

1. Row context menu in some cases is clipped
2. There are two scrolls in the list - one for the whole list and one for the "letter group"

https://user-images.githubusercontent.com/41268731/189847637-1ccc93ba-8e62-4ee8-87e3-3d279f3e4e4f.mov

**Comment by wercix** [comment]
@igorbezsmertnyi could you please close this draft?

**Comment by wojcik92michal** [comment]
Handled by another PR, https://github.com/mudita/mudita-center/pull/977

**Pull request closed without merging**